### PR TITLE
mp++ recipe

### PR DESCRIPTION
**Pull request by bluescarni**

### CODE REVIEW

**Comment by isuruf** [comment]
Hmm, conda-smithy created the feedstock as https://github.com/conda-forge/mp--feedstock. @ocefpaf, can you rename that to `mp++-feedstock`?

**Comment by isuruf** [comment]
@bluescarni, does `conda install mp++ --use-local` work?

**Comment by ocefpaf** [comment]
Yeah. I just noticed that... It is getting late here I will try to fix this tomorrow.

File: recipes/mp++/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set version = "0.2" %}
+
+package:
+  name: mp++
```
Comment (jakirkham) [inline recipes/mp++/meta.yaml]: Hmm...so there seem to be some issues here.

First is that `+` is not permissible in a repo name. However it seems GitHub doesn't fail, it just changes `++` to `-`. Second is whether `+` is permitted in `conda` package names. Presumably `+` works ok with `conda` as the tests passed. However I'm not sure whether that is allowed in a package name or not (e.g. noted in the docs).

Currently the first issue is blocking conversion of this recipe. So we should probably decide how we want to handle that. What are your thoughts on just calling it `mppp`? Given it matches the repo name, maybe this is ok. Do you have other thoughts on naming?

Comment (jakirkham) [inline recipes/mp++/meta.yaml]: It sounds like `+` is also something not explicitly allowed by `conda` (at least according to the docs).

> The name of a package, without any reference to a particular version. Conda package names are normalized, and may contain only lowercase alpha characters, numeric digits, underscores, hyphens, or dots. In usage documentation, these will be referred to by package_name.

ref: https://conda.io/docs/building/pkg-name-conv.html

**Comment by bluescarni** [comment]
@isuruf @ocefpaf @jakirkham Thanks for all the help!

I have no issues renaming the conda package to ``mppp``, how should I proceed about doing it?

I will try to do it later today as soon as possible.

**Comment by isuruf** [comment]
Send a PR changing the folder name and the package name. Then we can merge it

**Comment by bluescarni** [comment]
If I have understood correctly, I should open a new pr with the new name? Or should I commit to this pr?

**Comment by isuruf** [comment]
Create a new PR

**Comment by jakirkham** [comment]
Went ahead and proposed this in PR ( https://github.com/conda-forge/staged-recipes/pull/3009 ). Would appreciate if people double checked that this is correct.

**Pull request merged**